src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/translator_cache_dir_disabled.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config secret="s3cr3t">
+        <framework:translator enabled="true" fallback="fr" logging="true" cache_dir=""/>
```

Comment:
`cache-dir="null"`

Reply:
Fixed:
I wanted to test the empty string.
what do you think about ?
Status: Waiting Feedback

Reply:
What would a use case for using the empty string?

Reply:
I wanted to confirm, with a test the "Magically try". 
But now it is not useful
thank's for your feedback :+1: